docs/canonicalk8s/charm/howto/install/install-terraform.md

```diff
@@ -7,7 +7,7 @@ the [Terraform Juju Provider][juju-provider-tf].
 
 This guide requires the following:
 
-- A Juju controller and model
+- A Juju controller
 <!-- TODO remove Juju prerequisites once ground up module is available -->
```

Comment:
Just double checking this ground up module is not available yet?

Reply:
I don't know what this comment means.

Reply:
Maybe @louiseschmidtgen will know?